docs/troubleshooting.md

```diff
@@ -0,0 +1,8 @@
+## Conflicting Package Names
+
+The Marketplace operator does not support reusing package names between two Operator Sources. To avoid unexpected behavior users should avoid adding any operator sources that reuse existing package names.
+
+Users can view existing package names with the following command:
+```bash
+$ oc get packagemanifests
```

Comment:
Should this use the marketplace namespace in order to list the packages available via operatorsources?
```bash
$ oc get packagemanifests -n marketplace
```
Although, maybe it should use `--all-namespaces` to list all existing packages on cluster since some may be installed via CatalogSource directly

Reply:
Changed to `--all-namespaces`.